gradoop-store/gradoop-hbase/src/main/java/org/gradoop/storage/impl/hbase/io/inputformats/EdgeTableInputFormat.java

```diff
@@ -20,25 +20,20 @@
 import org.apache.hadoop.hbase.client.Result;
 import org.apache.hadoop.hbase.client.Scan;
 import org.apache.hadoop.hbase.filter.FilterList;
-import org.gradoop.common.model.api.entities.EPGMEdge;
-import org.gradoop.common.model.api.entities.EPGMVertex;
+import org.gradoop.common.model.impl.pojo.Edge;
 import org.gradoop.storage.impl.hbase.api.EdgeHandler;
 import org.gradoop.storage.impl.hbase.constants.HBaseConstants;
 import org.gradoop.storage.impl.hbase.filter.HBaseFilterUtils;
 
 /**
  * Reads edge data from HBase.
- *
- * @param <V> EPGM vertex type
- * @param <E> EPGM edge type
  */
-public class EdgeTableInputFormat<E extends EPGMEdge, V extends EPGMVertex>
```

Comment:
Duplicate code ::getScanner

Reply:
I can not outsource the `getQuery()` api definition from `EdgeHandler` to `ElementHandler` because of the generics definition of `HBaseElementFilter<Edge/Vertex/GraphHead>`. Or do you see a solution?

Reply:
Create a BaseInputFormat and adding a protected function with generic type

Reply:
Thx!